CONTRIBUTING.md

```diff
@@ -7,6 +7,8 @@
 * Commit, do not mess with rakefile, version, or history.
   (if you want to have your own version, that is fine but bump version in a commit by itself I can ignore when I pull)
 * Send a pull request. Bonus points for topic branches.
+  * Your branch must pass tests: run `rake` or `rspec spec/{file or functionality you modified}_spec.rb`.
+  * Your branch must pass style checking: run `brew style homebrew/bundle` if your working copy is tapped. If not, run `brew style .` and ignore "Use hash rockets syntax" warnings.
```

Comment:
How about `Your PR must pass Rubocop checks: run brew style homebrew/bundle`. I'd suggest omitting the `brew style` one because that will generally be wrong.